src/controllers/applicationController.ts

```diff
@@ -81,7 +110,11 @@ export class ApplicationController implements ApplicationControllerInterface {
 		}
 
 		try {
+			// Save the new application
 			await this._applicantService.save(newApplication, cvFile);
+
+			// Remove the partial application
+			await this._partialApplicantService.remove(reqUser.id);
```

Comment:
There are a few other places where using transactions is very much required to prevent inconsistencies. I made an issue for this (#47) since it would need the same transaction to be available in multiple services.

I can either do a quick fix for this to pass the transaction to each of the services or I can mark this function as something that needs attention when doing #47 (which I intend to work on next).

Let me know which approach you would prefer :)

Reply:
Making a note in #47 is fine in that case!